klarna-on-demand/build.gradle

```diff
@@ -40,4 +42,77 @@ dependencies {
     androidTestCompile 'org.easymock:easymock:3.3'
 
     compile fileTree(dir: 'libs', include: ['*.jar'])
+}
+
+// Add tasks for javadoc and source jar generation
+task androidJavadocs(type: Javadoc) {
```

Comment:
From what I read, as long as you bundle it along with your package in Maven you are doing the "right" thing (we do that, in case you were wondering).